source/numerics.tex

```diff
@@ -12407,6 +12420,7 @@
 
 \rSec3[linalg.conj.conjugated]{Function template \tcode{conjugated}}
 
+\indexlibraryglobal{conjugated}%
```

Comment:
Is there a reason to use this, instead of using `@\libglobal{conjugated}@` in the actual declaration below?